vm/src/hint_processor/builtin_hint_processor/bootloader_hints.rs

```diff
@@ -261,6 +261,23 @@ pub fn import_packed_output_schemas() -> Result<(), HintError> {
     Ok(())
 }
 
+/// Implements %{ isinstance(packed_output, PlainPackedOutput) %}
+///
+/// Stores the result in the `ap` register to be accessed by the program.
+pub fn is_plain_packed_output(
+    vm: &mut VirtualMachine,
+    exec_scopes: &mut ExecutionScopes,
+) -> Result<(), HintError> {
+    let packed_output: PackedOutput = exec_scopes.get(vars::PACKED_OUTPUT)?;
+    let result = match packed_output {
+        PackedOutput::Plain(_) => 1,
+        _ => 0,
+    };
+    insert_value_into_ap(vm, result)?;
```

Comment:
It's not clear to me why this would be written here, maybe this is a chance for you to school me :)

When I see `%{ isinstance(packed_output, PlainPackedOutput) %}` I think "isinstance returns a temporary onto the stack which is dropped before returning from the hint to the VM."

If you're clear on why this should write the value to the VM, can you explain what's wrong with my thinking?

Reply:
Nevermind, I'm clear about this now, I had the wrong expectation of `isinstance()`.